testing/pypy/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor:Mihnea Saracin <mihnea.saracin@rinftech.com>
+# Maintainer:Mihnea Saracin <mihnea.saracin@rinftech.com>
+pkgname=pypy
+pkgver=5.7.1
+pkgrel=0
+pkgdesc="pypy2"
```

Comment:
A better package description would be nice. Maybe something like this: `pkgdesc="A fast, compliant alternative implementation of the Python 2 language"`.